provision.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+echo "updating system packages"
 sudo apt update
 
-sudo apt install -y nginx
\ No newline at end of file
+echo "install nginx"
+sudo apt install -y nginx
+
+echo 'copy website to server'
+sudo cp -r /vagrant/website /var/www/website/
+
+echo "copy website.config file to sites-enabled nginx"
+sudo cp /vagrant/website.config /etc/nginx/sites-enabled/website.config
+
+echo "restart the nginx service"
+sudo systemctl reload nginx
+
+echo "display device configuration"
+sudo ifconfig
+
+echo "display disk usage"
+sudo df -h
+
+echo "display website html"
+sudo curl localhost:81
```

Comment:
This is the wrong address. Should be something like "IP of vm:81" by getting the IP from the Vagrantfile so `curl 192.168.56.10:81`.

Also, you should only use sudo on commands that say they cannot run without privilege. Curl does not need sudo to run.